### PR TITLE
Story3/task/static page

### PR DESCRIPTION
**Pull request by alecorra**
craeted the static pages with css and placeholders for the galleries

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+.bone {
+    width: 50%;
+    position: absolute;
+    left: 26%;
+    z-index: -1;
+}
+
+.header {
+    position: relative;
+
```
Comment (neilbulloch) [inline CSS/style.css:22]: white space : )

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                <a class="dropdown-item" href="#">Action</a>
+                <a class="dropdown-item" href="#">Action</a>
+                <a class="dropdown-item" href="#">Action</a>
+                <a class="dropdown-item" href="#">Action</a>
+                <a class="dropdown-item" href="#">Action</a>
+                <a class="dropdown-item" href="#">Action</a>
+
+            </div>
+        </div>
+
```
Comment (neilbulloch) [inline index.html:39]: white space

**Review by neilbulloch — CHANGES_REQUESTED** [review]
could be more responsive in the future but not important for now

File: CSS/style.css (hunk 1 above)
Comment (alecorra) [reply CSS/style.css:22]: done

File: index.html (hunk 2 above)
Comment (alecorra) [reply index.html:39]: done

**Pull request merged**